### PR TITLE
 SUPPORTED_APPS

### PR DESCRIPTION
**Pull request by corgan2222**
- nsfminer
- shinobipro shinobi-face

grafik

### CODE REVIEW

**Comment by corgan2222** [comment]
I hope this time the PR is correct. :)

File: src/gpustat/usr/local/emhttp/plugins/gpustat/gpustatus.page [hunk 1]
```diff
@@ -183,6 +183,12 @@ Icon="gpustat.png"
                 <span id="gpu-img-span-deepstack" class='gpu-img-span gpu-img-span-deepstack'>
                     <img id='gpu-deepstack' class='gpu-image' src="/plugins/gpustat/images/deepstack.png">
                 </span>
+                <span id="gpu-img-span-nsfminer" class='gpu-img-span gpu-img-span-nsfminer'>
+                    <img id='gpu-nsfminer' class='gpu-image' src="/plugins/gpustat/images/NsfminerOC.png">
```
Comment (b3rs3rk) [inline src/gpustat/usr/local/emhttp/plugins/gpustat/gpustatus.page:187]: Need to lowercase the name for this! NsfminerOC.png -> nsfmineroc.png -- you will need to change the file name as well and push that commit.  PR should update automatically and then I can merge.

**Review by b3rs3rk — CHANGES_REQUESTED** [review]
Review comments, just need the file for nsfminer lowercased

File: src/gpustat/usr/local/emhttp/plugins/gpustat/gpustatus.page (hunk 1 above)
Comment (corgan2222) [reply src/gpustat/usr/local/emhttp/plugins/gpustat/gpustatus.page]: fixed :)

**Comment by corgan2222** [comment]
Should be fixed.

**Review by b3rs3rk — APPROVED** [review]
Looks good!  Thanks for the contribution.  I'll try to get a new release out in the next few days.

**Pull request merged**